src/rp2_common/pico_cyw43_driver/btstack_hci_transport_cyw43.c

```diff
@@ -148,6 +151,13 @@ static void hci_transport_cyw43_process(void) {
         } else {
             has_work = false;
         }
+// PICO_CONFIG: PICO_BTSTACK_CYW43_MAX_HCI_PROCESS_LOOP_COUNT, limit the max number of iterations of the hci processing loop, type=int, advanced=true, group=pico_btstack
+#ifdef PICO_BTSTACK_CYW43_MAX_HCI_PROCESS_LOOP_COUNT
+        loop_count++;
+        if (loop_count > PICO_BTSTACK_CYW43_MAX_HCI_PROCESS_LOOP_COUNT) {
```

Comment:
seems like this should be >=

Reply:
Or I guess both lines could be squashed to:
```c
if (loop_count++ > PICO_BTSTACK_CYW43_MAX_HCI_PROCESS_LOOP_COUNT) {
```
?